src/Bluesky.NET/ApiClients/BlueskyApiClient.cs

```diff
@@ -16,19 +16,22 @@ namespace Bluesky.NET.ApiClients;
 public partial class BlueskyApiClient : IBlueskyApiClient
 {
     private readonly HttpClient _httpClient = new();
+    private string _baseUrl = "https://bsky.social";
 
-    public async Task<Result<AuthResponse>> RefreshAsync(string refreshToken)
+    public async Task<Result<AuthResponse>> RefreshAsync(string refreshToken, string baseUrl = UrlConstants.BlueskyBaseUrl)
     {
-        var refreshUrl = $"{UrlConstants.BlueskyBaseUrl}/{UrlConstants.RefreshAuthPath}";
+        _baseUrl = baseUrl;
+        var refreshUrl = $"{baseUrl}/{UrlConstants.RefreshAuthPath}";
```

Comment:
Recommend making the following changes:
- change new parameter to `string? baseUrl = null`. Make sure to apply this change in the interface as well.
- Then on line 23, do this `_baseUrl = baseUrl ?? UrlConstants.BlueskyBaseUrl;`
- Then make sure on line 24 to use the correct variable: `{_baseUrl}/{UrlConstants.RefreshAuthPath}`